public/js/modules/nav.js

```diff
@@ -0,0 +1,123 @@
+import {
+  disableBodyScroll,
+  enableBodyScroll,
+} from "../lib/bodyScrollLock.esm.js";
+
+export default class nav {
+  constructor(el) {
+    this.el = el;
+    this.setVars();
+    this.bindEvents();
+  }
```

Comment:
If you haven't seen this pattern yet, I'm loosely following the now-older [Viget Modules](https://www.viget.com/articles/how-does-viget-javascript/) pattern here (replaced by Stimulus as default somewhere in [late 2021/early 2022?](https://www.viget.com/articles/stimulus-controllers-and-viget-modules-how-do-they-compare/))